accounts/abi/bind/backends/simulated.go

```diff
@@ -502,22 +502,34 @@ func (fb *filterBackend) GetLogs(ctx context.Context, hash common.Hash) ([][]*ty
 }
 
 func (fb *filterBackend) SubscribeNewTxsEvent(ch chan<- core.NewTxsEvent) event.Subscription {
-	return event.NewSubscription(func(quit <-chan struct{}) error {
-		<-quit
-		return nil
-	})
+	return nullSubscription()
```

Comment:
This looks really weird. Why do we have this function?

Reply:
We do this because returning a `nil` subscription is not allowed. `nullSubscription` returns a subscription that never sends anything.